asf_search/ASFProduct.py

```diff
@@ -24,6 +24,11 @@ def __init__(self, args: dict = {}, session: ASFSession = ASFSession()):
         self.baseline = translated['baseline']
         self.session = session
 
+        if 'additionalUrls' not in self.properties or len(self.properties['additionalUrls']) == 0:
```

Comment:
Let's make this a method so if users modify `additionalUrls` with something extra (like say, some random extra product from the umm that asf-search doesn't grab automatically) they can still download it when specifying ADDITIONAL/ALL_FILES